app/services/notify/notify-email.service.js

```diff
@@ -53,11 +53,15 @@ async function _sendEmail(notifyClient, templateId, emailAddress, options) {
       statusText: response.statusText.toLowerCase()
     }
   } catch (error) {
-    return {
+    const formattedError = {
       status: error.status,
       message: error.message,
       errors: error.response.data.errors
     }
+
+    global.GlobalNotifier.omfg('Notify send email failed', null, formattedError)
+
+    return formattedError
```

Comment:
Unfortunately, this won't work with how `omfg()` is formulated in `app/lib/base-notifier.lib.js`.

It expects something of type 'Error' for the 3rd argument. What you are providing is data, so should be passed as the second argument.

It is fine to call `omfg()` without an actual error, in which case you just leave the third argument off.

```suggestion
    const errorDetails = {
      status: error.status,
      message: error.message,
      errors: error.response.data.errors
    }

    global.GlobalNotifier.omfg('Notify send email failed', errorDetails)

    return errorDetails
```

This will apply to all the other places you are calling `omfg()` in this way.